jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -131,15 +132,34 @@ private void executeTimerJobInstance(TimerJobInstance timerJobInstance) throws E
     }
 
     private void recoverTimerJobInstance(EjbTimerJob ejbTimerJob, Exception e) {
+        if(isSessionNotFound(e)) {
```

Comment:
Just nitpicking about the format, if you can add a blank before parenthesis
```suggestion
        if (isSessionNotFound(e)) {
```

Reply:
done